tortoise/backends/aioodbc/executor.py

```diff
@@ -0,0 +1,67 @@
+from pypika import Parameter, functions
+from pypika.terms import Criterion, Field
+
+from tortoise import Model
+from tortoise.backends.base.executor import BaseExecutor
+from tortoise.fields import BigIntField, IntField, SmallIntField
+from tortoise.filters import (
+    contains,
+    ends_with,
+    insensitive_contains,
+    insensitive_ends_with,
+    insensitive_exact,
+    insensitive_starts_with,
+    starts_with,
+)
+
+
+def aiodbc_contains(field: Field, value: str) -> Criterion:
+    return field.like(f"%{value}%")
```

Comment:
Is the issue with these that you need to know what varchar length these needs to be cast to?
Now that I think of it, why are we doing the casting on some of these? (I think this behaviour predates me)

Is it to protect our users against doing stupid things like doing string-comparisons on number types? Surely we should just let it crash? :thinking:

Reply:
I investigated on the castings, and they were there from well before I got involved. I think that the code changed enough that it isn't needed anymore. Removed it, and CI passes. So we can hopefully remove most of these overrides.

Reply:
Score! I took quite a while trying to figure out how to bring in the length. I had figured this would just be a workaround, but sounds like it's fine as is :).